env/agentcompass/tests/test_evaluation_context.py

```
@@ -0,0 +1,154 @@
+
+
+class FakeManager:
+    job_id = "job-test"
+
+    def __init__(self):
+        self.rows = [{"record_id": "row-0", "job_id": "job-test", "session_id": "session-test", "step_id": 0, "is_terminal": False, "meta_json": {"eval": {"existing": True}}}]
+        self.inserted = []
+
+    async def init(self): pass
+    async def list_session_steps(self, *args, **kwargs): return list(self.rows)
+    async def insert_session_step_rows(self, rows): self.inserted.extend(rows); return [row["record_id"] for row in rows]
```

Ruff (0.16.3)

[error] Multiple statements on one line (semicolon)

(E702)